### ISSUE TITLE
The size of tensor a (8) must match the size of tensor b (64) at non-singleton dimension 0

### ISSUE THREAD

**Issue by rendez2k** [pharmapsychotic/clip-interrogator-ext#59]
Getting this error, have tried a clean install. Any ideas?

**Comment by ELIYASOUND** [comment]
same...

**Comment by swumagic** [comment]
me,too,

**Comment by ansonphong** [comment]
Yes this error is happening constantly, I'm unable to get interogator to work at all. On the latest Automatic1111 and everything else seems to be working otherwise.

**Comment by jgilfoil** [comment]
This is with release v1.6.0-RC of Automatic1111

**Comment by kishore25kumar** [comment]
Looks like issue with latest transformers, https://github.com/vladmandic/automatic/issues/108#issuecomment-1509056131

**Comment by hben35096** [comment]
In version 1.6, Negative prompt can be generated normally, but other options are not available.

**Comment by hben35096** [comment]
I think I found the cause and want to reinstall:
pip install 'transformers>=4.15.0,<=4.26.1'
For the requirements .txt and requirements_versions.txt of WebUI v1.6, change "transformers==4.30.2" to "transformers==4.26.1" (installed version) in the file

**Comment by hben35096** [comment]
> > 我想我找到了原因并想重新安装：pip install'transformers>=4.15.0，<=4.26.1' 对于WebUI v1.6的要求.txt和requirements_versions.txt，请在文件中将“transformers==4.30.2”更改为“transformers==4.26.1”（已安装版本）
> 
> 我似乎已经有了“变压器==4.26.1”，但我仍然收到错误。
> 
> 我还安装了“变压器==4.30.2”并得到了同样的错误。

Don't start WebUI after installing "transformers==4.26.1", wait until you change two "requirements .txt", and then launch, do you do this?

**Comment by Grocel** [comment]
Also has been reported at https://github.com/pharmapsychotic/clip-interrogator-ext/issues/64

**Comment by pharmapsychotic** [comment]
Extension is updated now, please update in the webui and restart and should be working again. Let me know if you have any issues!

**Comment by Grocel** [comment]
It's fixed for me now, thanks.
